src/Features/Core/Diagnostics/Analyzers/RudeEditUserDiagnosticAnalyzer.cs

```diff
@@ -24,7 +24,7 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         public override Task AnalyzeSyntaxAsync(Document document, Action<Diagnostic> addDiagnostic, CancellationToken cancellationToken)
         {
             // No syntax diagnostics produced by the EnC engine.  
-            return Task.FromResult(false);
+            return SpecializedTasks.EmptyTask;
```

Comment:
I reviewed all of the callers; they all just await the returned task and don't pay any attention to object identity, don't downcast, etc.